lib/src/room.dart

```diff
@@ -60,15 +66,9 @@ abstract class Room implements Listenable {
   /// Sends the provided message to the Agent.
   ///
   /// If the application does not support messaging, this will throw an exception.
+  @Deprecated('This function was moved into [MessagingClient].')
   Future<void> sendMessage(String text);
```

Comment:
I kept this function here as a Deprecated function because this would be a breaking change on the version which were distributed until now.